web/packages/core/src/plugin-polyfill.ts

```diff
@@ -46,7 +46,9 @@ class RuffleMimeTypeArray implements MimeTypeArray {
     }
 
     item(index: number): MimeType {
-        return this.__mimeTypes[index]!;
+        // This weird modulo behavior is done to emulate
+        // the behavior of browsers.
+        return this.__mimeTypes[index % 2 ** 32]!;
```

Comment:
I wonder if this would be more accurate as `index>>>0` to more closelt match the 32-bit int behavior.

Reply:
Hmm... testing this out, `navigator.plugins[-(2**32) + 1]` returns `undefined`, but if it behaved how you suggest, it would return the second element of the PluginArray.

Reply:
`navigator.plugins.items[-(2**32) + 1]` returns the second element on my end -- raw access of the array `navigator.plugins[2**32]` does not work

Reply:
Ah, makes sense! 👍